pkg/filesystem/subvolume.go

```diff
@@ -195,11 +244,11 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 					// check the state of the stale subvolume
 					// if it is snapshot-retained then skip listing it.
 					if state == "snapshot-retained" {
```

Comment:
Can we also use this var here?
```suggestion
					if state == snapshotRetained {
```